cosmic-usage-api/src/test/java/com/github/missioncriticalcloud/cosmic/api/usage/services/UsageCalculatorIT.java

```diff
@@ -1,42 +1,29 @@
 package com.github.missioncriticalcloud.cosmic.api.usage.services;
 
-import static com.github.missioncriticalcloud.cosmic.api.usage.repositories.es.MetricsEsRepository.DOMAIN_UUID_FIELD;
-import static com.github.missioncriticalcloud.cosmic.api.usage.repositories.es.MetricsEsRepository.RESOURCE_TYPE_FIELD;
-import static com.github.missioncriticalcloud.cosmic.api.usage.repositories.es.MetricsEsRepository.RESOURCE_UUID_FIELD;
-import static com.github.missioncriticalcloud.cosmic.api.usage.repositories.es.MetricsEsRepository.TIMESTAMP_FIELD;
 import static com.github.missioncriticalcloud.cosmic.usage.core.utils.FormatUtils.DATE_FORMATTER;
 import static org.assertj.core.api.Assertions.assertThat;
 
 import java.io.IOException;
 import java.math.BigDecimal;
 import java.util.List;
 
-import com.github.missioncriticalcloud.cosmic.api.usage.exceptions.NoMetricsFoundException;
+import com.github.missioncriticalcloud.cosmic.usage.core.exceptions.NoMetricsFoundException;
 import com.github.missioncriticalcloud.cosmic.usage.core.model.Compute;
 import com.github.missioncriticalcloud.cosmic.usage.core.model.Domain;
 import com.github.missioncriticalcloud.cosmic.usage.core.model.Networking;
 import com.github.missioncriticalcloud.cosmic.usage.core.model.Report;
 import com.github.missioncriticalcloud.cosmic.usage.core.model.Unit;
 import com.github.missioncriticalcloud.cosmic.usage.core.model.Usage;
-import com.github.missioncriticalcloud.cosmic.usage.core.model.types.ResourceType;
+import com.github.missioncriticalcloud.cosmic.usagetestresources.EsTestUtils;
```

Comment:
I'd move `EsTestUtils` to a package called `com.github.missioncriticalcloud.cosmic.usage.testresources`.